app/models/ems_refresh/save_inventory_physical_infra.rb

```diff
@@ -115,11 +115,19 @@ def save_asset_detail_inventory(parent, hash)
   end
 
   #
-  # Saves the drive information of a storage
+  # Saves the disks information of a storage
   #
-  def save_physical_disk_inventory(parent, hash)
-    return if hash.nil?
-    save_inventory_single(:physical_disk, parent, hash)
+  def save_physical_disks_inventory(physical_storage, hashes, target = nil)
+    return if hashes.nil?
+    target = physical_storage if target.nil?
+    deletes = target == physical_storage ? :use_association : []
```

Comment:
Actually since these are only under a physical storage, there is no case where you wouldn't want to `:use_association`
This is usually meant for top level collections (e.g. if you had ems.physical_disks and you target e.g. a physical_host not the full ems).

Just take this out and pass `:use_association` to `save_inventory_multi`

Reply:
@agrare Ok, got it, I changed it to pass `:use_association` directly to `save_inventory_multi`.